src/main/telemetry/hott.c

```diff
@@ -342,6 +342,10 @@ void initHoTTTelemetry(void)
     portConfig = findSerialPortConfig(FUNCTION_TELEMETRY_HOTT);
     hottPortSharing = determinePortSharing(portConfig, FUNCTION_TELEMETRY_HOTT);
 
+#if defined (USE_HOTT_TEXTMODE) && defined (USE_CMS)
+    hottDisplayportRegister();
```

Comment:
cmsDisplayPortRegister(displayPortHottInit());

Reply:
I don't want a CMS dependency in hott.c, because the "display" shouldn't know CMS and CMS  shouldn't know its displays directly. Both should only know displayport. 
Displayport is the interface to CMS, so all input/output goes through it.